.env.base

```diff
@@ -39,6 +40,10 @@ REACT_APP_FEATURE_OSMOSIS_SWAP=true
 REACT_APP_FEATURE_THOR_SWAP=true
 REACT_APP_FEATURE_ZRX_SWAP=true
 
+# chat woot
+REACT_APP_CHATWOOT_TOKEN=jmoXp9BPMSPEYHeJX5YKT15Q
+REACT_APP_CHATWOOT_URL=https://app.chatwoot.com
```

Comment:
now we have this we can use it in the CSP. ty for moving the token into env vars